src/Apps/Artwork/artworkRoutes.tsx

```diff
@@ -33,16 +31,8 @@ export const artworkRoutes: RouteProps[] = [
       const requestError = (props as any).artworkResult?.requestError
       const requestErrorStatusCode = requestError?.statusCode
 
-      const enableCustomErrorPage = getFeatureFlag(
-        "onyx_custom_artwork_error_page"
-      )
-
       if (requestErrorStatusCode) {
-        if (enableCustomErrorPage) {
-          updateContext("statusCode", requestErrorStatusCode)
-        } else {
-          throw new HttpError(requestErrorStatusCode)
```

Comment:
don't you need to update where we handle these errors to not to anymore?

Reply:
I did — it just does what the flag was enabling now, which is apparently putting the error code into async local storage.